gammapy/maps/geom.py

```diff
@@ -308,6 +308,29 @@ def __init__(self, nodes, interp="lin", name="", node_type="edges", unit=""):
 
         self._nbin = nbin
 
+    def is_aligned(self, other, atol=2e-2):
+        """Check if other map axis is aligned.
+
+        Two axes are aligned if their center coordinate values map to integers
```

Comment:
Testing only on centers does not guarantee that the test will work. Imagine the following situation:
```
ax1=MapAxis([1,2,3], interp='lin', node_type='edges')                                                                                    
ax2=MapAxis([1.5,2.5], interp='log', node_type='center')   
```
These two axes would be considered aligned although their edges are definitely not. One should test that `interp_mode` is the same. I suppose  this is sufficient.

Reply:
Thanks, I'll address this in a follow up PR...

Reply:
Fixed in master: https://github.com/gammapy/gammapy/commit/c0e58f48fdb990256d3b1e683200f7649b300f0e